src/libraries/System.Private.Xml/src/System/Xml/Cache/XPathNode.cs

```diff
@@ -124,7 +125,11 @@ public XPathDocument Document
         /// </summary>
         public string BaseUri
         {
-            get { return _info.BaseUri; }
+            get
```

Comment:
should this return nullable instead?

Reply:
This property is referenced by `XPathNavigator.BaseURI` which is also non-nullable. Based on the docs, it should never return null but `string.Empty`.

https://docs.microsoft.com/en-us/dotnet/api/system.xml.xpath.xpathnavigator.baseuri?view=netcore-3.1
> The location from which the node was loaded, or Empty if there is no value.

Reply:
consider `_info.BaseUri ?? string.Empty`